crates/bermuda/src/lib.rs

```diff
@@ -58,35 +68,97 @@ fn triangulate_path_edge(
         limit.unwrap_or(3.0),
         bevel.unwrap_or(false),
     );
-    let triangle_data: Vec<u32> = result
+    triangulation_to_edge_triangulate(py, &result)
+}
+
+fn triangulation_to_edge_triangulate(
+    py: Python<'_>,
+    data: &PathTriangulation,
+) -> PyEdgeTriangulation {
+    let triangle_data: Vec<u32> = data
         .triangles
         .iter()
         .flat_map(|t| [t.x as u32, t.y as u32, t.z as u32])
         .collect();
 
-    // Convert back to numpy array ((M-2)x3) if triangles is not empty, otherwise create empty array (0x3).
-    let triangle_array = if !result.triangles.is_empty() {
-        PyArray::from_vec(py, triangle_data).reshape([result.triangles.len(), 3])?
+    let triangle_array = if !data.triangles.is_empty() {
+        PyArray::from_vec(py, triangle_data).reshape([data.triangles.len(), 3])?
     } else {
         PyArray2::<u32>::zeros(py, [0, 3], false)
     };
 
-    let flat_centers: Vec<f32> = result.centers.iter().flat_map(|p| [p.x, p.y]).collect();
-    let flat_offsets: Vec<f32> = result.offsets.iter().flat_map(|v| [v.x, v.y]).collect();
+    let flat_centers: Vec<f32> = data.centers.iter().flat_map(|p| [p.x, p.y]).collect();
+    let flat_offsets: Vec<f32> = data.offsets.iter().flat_map(|v| [v.x, v.y]).collect();
 
     Ok((
         PyArray::from_vec(py, flat_centers)
-            .reshape([result.centers.len(), 2])?
+            .reshape([data.centers.len(), 2])?
             .into(),
         PyArray::from_vec(py, flat_offsets)
-            .reshape([result.offsets.len(), 2])?
+            .reshape([data.offsets.len(), 2])?
             .into(),
         triangle_array.into(),
     ))
 }
 
+fn triangulation_to_face_triangulate(
```

Comment:
```suggestion
fn triangulation_to_face_triangulation(
```

Same rationale as above with additional of similar function docstring.

Reply:
changed name,